### PR TITLE
Add CVE-2017-5811

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by GeetikaBatra** [comment]
The reference links shows multiple CVE's associated. Do we need to worry about them?

File: database/java/2017/5811.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - http://www.securitytracker.com/id/1038407
+    - https://support.hpe.com/hpsc/doc/public/display?docId=emr_na-hpesbgn03740en_us
+
+affected:
+    - groupId: com.hpe.nv
+      artifactId: hpe-nv-dtos
+      version:
+        - "1.0.0"
+      fixedin:
+        - "!FixMe"
```
Comment (GeetikaBatra) [inline database/java/2017/5811.yaml]: Where do we get this info? Do we leave it as it is if this vulnerability is not fixed yet?

Comment (msrb) [reply database/java/2017/5811.yaml]: Exactly, if the vulnerability is not fixed yet, then we would remove the `fixedin` section completely. In this case, one of the [links in references](https://support.hpe.com/hpsc/doc/public/display?docId=emr_na-hpesbgn03740en_us) describes which versions have the vulnerabilities fixes ("RESOLUTION" section).

**Comment by msrb** [comment]
> The reference links shows multiple CVE's associated. Do we need to worry about them?

Nope, we only care about this particular CVE. What's happening here is that vendor (HP) is fixing multiple vulnerabilities in one of its products.

**Comment by msrb** [comment]
This doesn't look like a (open-source) Maven package to me. There is only handful of packages in Maven Central that could be related: http://repo1.maven.org/maven2/com/hpe/nv/
But all of them are related to "HPE Network Virtualization" not "HPE Network Automation".

Therefore I'd close this as FP.

**Comment by GeetikaBatra** [comment]
@msrb, ohh okay. We need to make sure that it is open source. Will keep that in mind. Closing this one for now.

**Pull request closed without merging**